opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp

```diff
@@ -402,4 +458,43 @@ class Co2GasPvt
 
 } // namespace Opm
 
+namespace Opm::gpuistl{
+    template<class Scalar, class Params, class GPUContainer>
+    Co2GasPvt<Scalar, Params, GPUContainer>
+    move_to_gpu(Co2GasPvt<Scalar> cpuCo2){
+        return Co2GasPvt<Scalar, Params, GPUContainer>(move_to_gpu<Scalar, std::vector<Scalar>, GPUContainer>(cpuCo2.getParams()),
+                                               GPUContainer(cpuCo2.getBrineReferenceDensity()),
+                                               GPUContainer(cpuCo2.getGasReferenceDensity()),
+                                               GPUContainer(cpuCo2.getSalinity()),
+                                               cpuCo2.getEnableEzrokhiDensity(),
+                                               cpuCo2.getEnableVaporization(),
+                                               cpuCo2.getActivityModel(),
+                                               cpuCo2.getGasType());
+    }
+
+    template <class Scalar, class InputParams, class OutputParams, class ContainerType, class ViewType>
+    Co2GasPvt<Scalar, OutputParams, ViewType>
+    make_view(const Co2GasPvt<Scalar, InputParams, ContainerType>& co2GasPvt) {
```

Comment:
Usually place the opening brace for functions, classes and structs on the next line. Much easier to see that it is a function. But for loops and ifs, place it on the same line. I thought the clang-format definition in the module did that, but I guess I was wrong.